python/cudf/cudf/tests/pytest.ini

```diff
@@ -3,3 +3,4 @@
 [pytest]
 markers =
     spilling: mark benchmark a good candidate to run with `CUDF_SPILL=ON`
+xfail_strict = true
```

Comment:
Can we do the same for other packages in the repo? I would expect that dask-cudf / custreamz / cudf-kafka / strings_udf won't have as many problems as cudf. This is fine to do in a follow-up PR, I just want testing conventions to be consistent in the repo.

Reply:
Opened an issue.